src/init/init.ts

```diff
@@ -73,16 +73,15 @@ function checkIsMinGW(): boolean {
  * read the description from its package.json.
  */
 function getGeneratorDescription(generator: YeomanEnvironment.GeneratorMeta, generatorName: string): GeneratorDescription {
-  const name = getDisplayName(generatorName);
   let description: string = '';
+  let genName = generatorName;
```

Comment:
I was trying to avoid calling `getDisplayName()` multiple times (maybe unnecessarily so?). I'm thinking of pulling out this `try-catch` block for improved readability, which would also result in `getDisplayName()` being called only once. What do you think?